MAUI/Scheduler/cell-selection-view.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: cell-selection
+title: Cell selection in the .NET MAUI Scheduler | Syncfusion
+description: Learn here all about how to customize the cell selection in Syncfusion .NET MAUI Scheduler (SfScheduler) control.
```

Comment:
description: Learn how to customize the cell selection in Syncfusion .NET MAUI Scheduler (SfScheduler) control here.

Reply:
I have completed the corrections